truss/base/trt_llm_config.py

```diff
@@ -43,7 +44,6 @@ class TrussTRTLLMQuantizationType(str, Enum):
 
 class TrussTRTLLMPluginConfiguration(BaseModel):
     paged_kv_cache: bool = True
-    gemm_plugin: str = "auto"
```

Comment:
This will break our examples / existing builds we may have for users, correct? We should maybe keep the parameter but remove it from taking effect and then show a deprecation warning.

Reply:
Removing it here, will drop the option silently. 

The parameter has never done anything. Only valid option was to run e.g. a bfloat16 weights in float16, which would have led to a warning and decreased performance.